ecc/bls12-377/fp/element_test.go

```diff
@@ -2313,7 +2325,7 @@ func (z *Element) matchVeryBigInt(aHi uint64, aInt *big.Int) error {
 
 	slice := append(z[:], aHi)
 
-	return field.BigIntMatchUint64Slice(&aIntMod, slice)
+	return bigIntMatchUint64Slice(&aIntMod, slice)
```

Comment:
Maybe we can start getting rid of some of the tests for subcomponents of the Pornin inverse algo not that it's been a while and it seems to work.

Reply:
yep, even more so that... the pornin Inverse has this "fallback" mechanism to call the inverseExp.